utils/build-script-impl

```diff
@@ -159,9 +159,9 @@ KNOWN_SETTINGS=(
     enable-llvm-assertions      "1"              "set to enable llvm assertions"
     build-llvm                  "1"              "set to 1 to build LLVM and Clang"
     build-swift-tools           "1"              "set to 1 to build Swift host tools"
-    build-swift-stdlib          "1"              "set to 1 to build the Swift standard library"
     build-swift-stdlib-unittest-extra "0"        "set to 1 to build optional StdlibUnittest components"
     build-swift-sdk-overlay     "1"              "set to 1 to build the Swift SDK overlay"
```

Comment:
I think that's still possible. The old parameters were:

```
--build-swift-stdlib=0 --build-swift-sdk-overlay=0
```

Whereas the new parameters are:

```
--build-swift-dynamic-stdlib=0 --build-swift-static-stdlib=0 --build-swift-sdk-overlay=0
```

These changes aren't meant to affect the ability to build just the compiler, they simply intend to split the `--build-swift-stdlib` into two separate parameters: `--build-swift-dynamic-stdlib` and `--build-swift-static-stdlib`.

On OS X, I've confirmed that `utils/build-script -R -- --build-swift-dynamic-stdlib=0 --build-swift-static-stdlib=0 --build-swift-sdk-overlay=0` completes successfully.

I wasn't able to successfully run `utils/build-script --build-swift-stdlib=0 --build-swift-sdk-overlay=0` on the latest commit on `master` on Linux, however. I think that set of parameters may not be working on Linux at the moment.